python_on_whales/components/container/cli_wrapper.py

```diff
@@ -1795,21 +1788,16 @@ def stop(
             time: Seconds to wait for stop before killing a container (default 10)
 
         # Raises
-            `python_on_whales.exceptions.NoSuchContainer` if the container does not exists.
+            `python_on_whales.exceptions.NoSuchContainer` if any of the
+            containers do not exist.
         """
         containers = to_list(containers)
-        if containers == []:
+        if len(containers) == 0:
```

Comment:
```suggestion
        if containers == []:
```
Same here, we don't get enough benefits to justify a small change like this. Otherwise we'd get pull requests with this kind of diff all the time.